topics/application-gradle-plugin.md

```diff
@@ -0,0 +1,68 @@
+[//]: # (title: Distribution via Gradle Application plugin)
+
+From the Gradle [documentation](https://docs.gradle.org/current/userguide/application_plugin.html):
+> The Application plugin facilitates creating an executable JVM application. 
+> It makes it easy to start the application locally during development, 
+> and to package the application as a TAR and/or ZIP including operating system specific start scripts.
+
+We will use the second responsibility of this plugin and show how to package a simple Ktor application.
+
+Let's start with the following project structure:
+
+```text
+├── build.gradle
+├── settings.gradle
+└── src
+    └── main
+        ├── kotlin
+        └── resources
+├── gradle
+├── gradlew
+└── gradlew.bat
+```
+You can use the [wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:adding_wrapper) task to generate `gradle`, `gradlew` and `gradlew.bat` files.
+
+Here is the initial content for the build configuration:
+
+<tabs>
+    <tab title="Gradle (Groovy)">
+        <code style="block" lang="Groovy" interpolate-variables="true" src="snippets/gradle-app/initial.gradle" />
+    </tab>
+</tabs>
+
+To use Ktor we need to add corresponding dependencies:
+
+<tabs>
+    <tab title="Gradle (Groovy)">
+        <code style="block" lang="Groovy" interpolate-variables="true" src="snippets/gradle-app/ktor-deps.gradle" />
+    </tab>
+</tabs>
+
+Then create a file with the simplest possible code for our server application:
```

Comment:
Then create a file .... 
-> 

Four our sample we're using a simple Ktor server that responds to a URL with some text: